microovn/ovn/environment/environment.go

```diff
@@ -17,7 +17,6 @@ import (
 	"github.com/canonical/lxd/shared/logger"
 	"github.com/canonical/microcluster/v2/cluster"
 	"github.com/canonical/microcluster/v2/state"
-
```

Comment:
Is removal of this blank line intentional?

It provided logical separation of imports, which I think we'd like to keep.

Reply:
Thanks for catching this. It wasn't intentional.

Reply:
nit: thanks for putting it back, it appears to be removed in commit 2 and put back in commit 5, so if you have an opportunity to move it to commit 2 that would give a cleaner git history.